sdk/storage/inc/blobs/internal/protocol/blob_rest_client.hpp

```diff
@@ -831,8 +830,14 @@ namespace Azure { namespace Storage { namespace Blobs {
       {
         auto request = Azure::Core::Http::Request(Azure::Core::Http::HttpMethod::Get, url);
         request.AddHeader("Content-Length", "0");
-        request.AddHeader("x-ms-version", options.Version);
-        request.AddHeader("x-ms-date", options.Date);
+        if (!options.Version.empty())
+        {
+          request.AddHeader("x-ms-version", options.Version);
+        }
+        else
+        {
+          request.AddHeader("x-ms-version", "2019-07-07");
```

Comment:
This needs to be a constant in the Storage SDK.

Reply:
@gilbertw Yes, absolutely. Actually every string literal is going to be a constant finally. But for now our time is very limited to meet the deadline and apparently this is not a blocker for the release.

Reply:
@gilbertw I created an [issue](https://github.com/Azure/azure-sdk-for-cpp/issues/164) to track this